MLOps/intro-to-ml-model-deployment-and-management/recipes/hydraulic-data.py

```diff
@@ -0,0 +1,147 @@
+"""Create hydraulic systems dataset"""
```

Comment:
This (same?) file exists in at least one other place, shared by another one of your PRs.

You may run into maintainability problems having different copies floating around.  It may be beneficial to refer to one of the already existing copies of this script instead of introducing it again.  IIRC, the other one may also have had some minor improvements gathered during the review.

Reply:
@orendain I will reference the hydraulic-data.py data recipe from the recipes repo.